concent_api/core/tests/test_auth_get_task_result.py

```diff
@@ -0,0 +1,964 @@
+import mock
+
+from django.test            import override_settings
+from django.urls            import reverse
+from freezegun              import freeze_time
+from golem_messages         import load
+from golem_messages         import message
+
+from core.models            import MessageAuth
+from core.tests.utils       import ConcentIntegrationTestCase
+from utils.testing_helpers  import generate_ecc_key_pair
+
+
+(CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY)                         = generate_ecc_key_pair()
+(DIFFERENT_PROVIDER_PRIVATE_KEY, DIFFERENT_PROVIDER_PUBLIC_KEY)   = generate_ecc_key_pair()
+(DIFFERENT_REQUESTOR_PRIVATE_KEY, DIFFERENT_REQUESTOR_PUBLIC_KEY) = generate_ecc_key_pair()
+
+
+def get_file_status_true_mock(_):
+    return True
+
+
+def get_file_status_false_mock(_):
+    return False
+
+
+@override_settings(
+    CONCENT_PRIVATE_KEY    = CONCENT_PRIVATE_KEY,
+    CONCENT_PUBLIC_KEY     = CONCENT_PUBLIC_KEY,
+    CONCENT_MESSAGING_TIME = 10,  # seconds
+    FORCE_ACCEPTANCE_TIME  = 10,  # seconds
+)
+class GetTaskResultIntegrationTest(ConcentIntegrationTestCase):
```

Comment:
This test name is already used elsewhere.

Reply:
:+1: